python/common/org/python/types/Dict.java

```diff
@@ -357,10 +357,12 @@ public void __delitem__(org.python.Object item) {
 
     @org.python.Method(
             __doc__ = "",
-            args = {"iterable"},
-            default_args = {"value"}
+            default_args = {"iterable", "value"}
```

Comment:
@cflee yes, it is because when I added iterable to args, and passed no params, it raised TypeError but error message doesn't match exactly with CPython's though they meant same, so I thought to handle `iterable=null` seperately. Should I add add test for it, or convert `iterable` back to args?